proto/select.proto

```diff
@@ -66,6 +66,14 @@ message SelectRequest {
 
 	// time zone offset in seconds
 	optional int64 time_zone_offset = 14;
+
+	/* flags is used to store flags that change the execution mode, it contains:
+		truncate_as_error = 1
+			Truncate wrong value will return error if set, return warning if not set.
+		...
+		add more when needed.
+	*/
+	optional uint64 flags = 15;
```

Comment:
The zero value behavior is different than current behavior, so keep it nullable is more compatible.